WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostActivity.java

```diff
@@ -1941,8 +1941,20 @@ public void onBackPressed() {
     public void onHistoryItemClicked(@NonNull Revision revision, @NonNull List<Revision> revisions) {
         AnalyticsTracker.track(Stat.REVISIONS_DETAIL_VIEWED_FROM_LIST);
         mRevision = revision;
+        ActivityLauncher.viewHistoryDetailForResult(this, mRevision, getRevisionsIds(revisions));
+    }
 
-        ActivityLauncher.viewHistoryDetailForResult(this, mRevision, revisions);
+    private long[] getRevisionsIds(@NonNull final List<Revision> revisions) {
```

Comment:
If I understand this correctly, number of returned ID's will be same as the number of revisions, so maybe we don't need to extract them into `idsList` first?

Can we simplify it into something like this (not tested)?:

```
  private long[] getRevisionsIds(@NonNull final List<Revision> revisions) {
        final long[] idsArray = new long[revisions.size()];
        for (int i = 0; i < revisions.size(); i++) {
            final Revision current = revisions.get(i);
            idsArray[i] = current.getRevisionId();
        }
        return idsArray;
    }
```